includes/in-person-payments/templates/html-in-person-payment-receipt.php

```diff
@@ -0,0 +1,189 @@
+<?php
+/**
+ * In Person Payments Receipt Template
+ *
+ * @package WooCommerce\Payments
+ */
+
+/**
+ * Helper to generate markup to render a price.
+ *
+ * @param  array  $product The product to display.
+ * @param  string $currency The currency to display.
+ * @return string
+ */
+function format_price_helper( array $product, string $currency ): string {
+	$active_price  = $product['price'];
+	$regular_price = $product['regular_price'];
+	$has_discount  = $active_price !== $regular_price;
+
+	if ( $has_discount ) {
+		return '<s>' . wc_price( $regular_price, [ 'currency' => $currency ] ) . '</s> ' . wc_price( $active_price, [ 'currency' => $currency ] );
+	}
+
+	return wc_price( $active_price, [ 'currency' => $currency ] );
+}
+
+?><!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Print Receipt</title>
+	<style>
+		body {
+			margin: 0;
+			padding: 0;
+			border: 0;
+		}
+
+		.align-left {
+			text-align: left;
+		}
+
+		.align-right {
+			text-align: right;
+		}
+		.align-top {
+			vertical-align: top;
+		}
+
+		.receipt {
+			min-width: 130px;
+			max-width: 300px;
+			margin: 0 auto;
+			text-align: center;
+			font-family: SF Pro Text, sans-serif;
+			font-size: 10px;
+
+		}
+
+		.receipt-table {
+			width: 100%;
+			border-collapse: separate;
+			border-spacing: 0 2px;
+		}
+
+		.receipt__header .title {
+			font-size: 14px;
+			line-height: 17px;
+			margin-bottom: 12px;
+			font-weight: 700;
+		}
+
+		.receipt__header .store {
+			padding: 0 12px;
+		}
+
+		.receipt__header .store__address {
+			margin-top: 12px;
+			line-height: 2px;
+		}
+
+		.receipt__header .store__contact {
+			margin-top: 4px;
+		}
+
+		.receipt__header .order__title {
+			font-weight: 800;
+		}
+
+		.receipt__transaction {
+			line-height: 2px;
+		}
+
+	</style>
+</head>
+<body>
+	<div class="receipt">
+		<div class="receipt__header">
+			<h1 class="title"><?php echo esc_html( $business_name ); ?></h1>
+			<hr />
+			<div class="store">
+				<?php if ( $support_address ) { ?>
+				<div class="store__address">
+					<p><?php echo esc_html( $support_address['line1'] ); ?></p>
+					<p><?php echo esc_html( $support_address['line2'] ); ?></p>
+					<p><?php echo esc_html( sprintf( '%s, %s %s %s', $support_address['city'], $support_address['state'], $support_address['postal_code'], $support_address['country'] ) ); ?></p>
```

Comment:
`implode(' ', [ $support_address['state'], $support_address['postal_code'], $support_address['country'] ] )` will cover the case if one part of address is not used in the merchant's country, so the output won't contain more whitespace than needed.

Reply:
Nice thanks!